### PR TITLE
GH-13: split web app storage into multiple entities

### LINKED ISSUE #13: split web app storage into multiple entities
we need to split storage up because now it so quite big, contain several logical entities and it is complicated to maintain it

### PR DESCRIPTION
**Pull request by alex-waverley**
Web app root storage contain 3 storages: videoPlayerStore, rosStore, webRTCStore
Closes #13

### CODE REVIEW

**Review by sskorol — CHANGES_REQUESTED** [review]

File: frontend/src/RosController/index.js [hunk 1]
```diff
@@ -63,6 +74,28 @@ class RosController {
       this.keyTopic.publish(keyMessage)
     }
   }
+
+  handleKeyboardShortcuts = throttle((e) => {
+    const keyName = e.key
+    if ([' ', 'spacebar'].includes(keyName)) {
+      this.onClick()
```
[the remark this answers is not in the document]
Comment (alex-waverley) [reply frontend/src/RosController/index.js]: done

File: frontend/src/RosController/index.js [hunk 2]
```diff
@@ -26,12 +37,12 @@ class RosController {
...
         messageType: TELEOP_MESSAGE_TYPE,
       })
-      this.teleopTopic.subscribe((msg) => this.setIsTeleopReady && this.setIsTeleopReady(msg.data))
+      this.teleopTopic.subscribe((msg) => this.setIsTeleopReady(msg.data))
     })
     this.ros.on('close', () => {
       console.warn('Disconnected from ROS bridge')
-      this.setIsWSConnected && this.setIsWSConnected(false)
-      this.setIsTeleopReady && this.setIsTeleopReady(false)
+      this.setIsWSConnected(false)
```
[the remark this answers is not in the document]
Comment (alex-waverley) [reply frontend/src/RosController/index.js:44]: done

File: frontend/src/RosController/index.js [hunk 3]
```diff
@@ -1,21 +1,32 @@
...
 class RosController {
+  @observable isTeleopReady = false
+  @observable isWSConnected = false
 
-  constructor(setIsWSConnected, setIsTeleopReady) {
-    this.setIsWSConnected = setIsWSConnected
-    this.setIsTeleopReady = setIsTeleopReady
+  constructor(rootStore) {
+    makeObservable(this)
+    this.rootStore = rootStore
```
[the remark this answers is not in the document]
Comment (alex-waverley) [reply frontend/src/RosController/index.js:16]: done (now it is used inside `handleKeyboardShortcuts` method)

**Review by sskorol — APPROVED** [review]
LGTM

**Pull request merged**